### PR TITLE
Technical mappings revision

### PR DESCRIPTION
**Pull request by idellal**

### CODE REVIEW

**Review by costezki — APPROVED** [review]

File: mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl [hunk 1]
```diff
@@ -875,8 +876,8 @@ tedm:ContractOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ContractOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ContractOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl:879]: Please double check this change. Seems a bit strange to use this org namespace for the instances that we create. Shouldn't it all stay in the "epd" namespace?

File: mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl [hunk 2]
```diff
@@ -1141,8 +1142,8 @@ tedm:MediatorOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/MediatorOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/MediatorOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl:1145]: Same comment as before. Please double check.

File: mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl [hunk 3]
```diff
@@ -1240,8 +1241,8 @@ tedm:ReviewProcedureInformationProviderOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ReviewProcedureInformationProviderOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ReviewProcedureInformationProviderOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl:1244]: Same comment as before. Please double check.

File: mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl [hunk 4]
```diff
@@ -1327,8 +1328,8 @@ tedm:ReviewerOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ReviewerOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ReviewerOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline mappings/package_F03/transformation/mappings/technical_mapping_F03.rml.ttl:1331]: Same comment as before. Please double check.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 5]
```diff
@@ -66,7 +67,7 @@ tedm:ContactPoint a rr:TriplesMap ;
 tedm:OrganisationAddress  a rr:TriplesMap ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/OrganisationAddress/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
+            rr:template "http://locn/ontology/OrganisationAddress/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:70]: Seems wrong. Please see my comments on a similar change in the copy of this file under the `package_F03/transformation` folder.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 6]
```diff
@@ -93,8 +94,8 @@ tedm:OrganisationChannel a rr:TriplesMap ;
 tedm:Organisation a rr:TripleMap ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/Organisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/Organisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:97]: Please double check this change. Seems a bit strange to use this org namespace for the instances that we create. Shouldn't it all stay in the "epd" namespace?

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 7]
```diff
@@ -571,7 +572,7 @@ tedm:ContractTermeLocationAddress a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ContractTermLocationAddress/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
+            rr:template "http://locn/ontology/ContractTermLocationAddress/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:575]: Same comment as before. Please double check.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 8]
```diff
@@ -875,8 +876,8 @@ tedm:ContractOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ContractOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ContractOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:879]: Same comment as before. Please double check.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 9]
```diff
@@ -1141,8 +1142,8 @@ tedm:MediatorOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/MediatorOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/MediatorOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:1145]: Same comment as before. Please double check.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 10]
```diff
@@ -1240,8 +1241,8 @@ tedm:ReviewProcedureInformationProviderOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ReviewProcedureInformationProviderOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ReviewProcedureInformationProviderOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:1244]: Same comment as before. Please double check.

File: src/mappings/technical_mapping_F03.rml.ttl [hunk 11]
```diff
@@ -1327,8 +1328,8 @@ tedm:ReviewerOrganisation a rr:TriplesMap ;
         ] ;
     rr:subjectMap
         [
-            rr:template "http://data.europa.eu/a4g/resource/ReviewerOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
-            rr:class epo:Organisation
+            rr:template "http://www.w3.org/ns/org/ReviewerOrganisation/{replace(replace(/TED_EXPORT/CODED_DATA_SECTION/NOTICE_DATA/NO_DOC_OJS, ' ', '-' ), '/' , '-')}/{unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw')}" ;
```
Comment (csnyulas) [inline src/mappings/technical_mapping_F03.rml.ttl:1331]: Same comment as before. Please double check.

**Comment by csnyulas** [comment]
Comments about the changes you made to clean up the prefixes

**You REMOVED:**
@prefix dc: <http://purl.org/dc/terms/> .		_GOOD (you have also updated its usage to 'dct')_
@prefix vann: <http://purl.org/vocab/vann/> .	
@prefix cc: <http://creativecommons.org/ns#> .
@prefix time: <http://www.w3.org/2006/time#> .
@prefix rml: <http://semweb.mmlab.be/ns/rml#> .		_GOOD (there is a second declaration, which you kept)_
@prefix ql: <http://semweb.mmlab.be/ns/ql#> .

**You ADDED:**
@prefix epd: <http://data.europa.eu/a4g/resource/> .	GOOD

**You have NOT REMOVED, but you SHOULD:**
@prefix :     <http://data.europa.eu/a4g/mapping/sf-rml/> .		_NOTE: There should not be any usage of it. If there is, change it to 'epd', 'tedm' or whatever makes most sense._
@prefix tedd: <http://data.europa.eu/a4g/dataset/sf-rml/> .		_NOTE: Change any usage if it to 'tedm'._


Regarding the removed prefixes 'vann', 'cc', 'time' and 'ql', you might or might not want to keep their declarations. They are definitely used (in prefixed form or full IRI form) in certain files in the `ted-sws-artefacts` repository, such as other validation files (e.g. `ePO_shacl_shapes.rdf`) that might be combined with the generated RDF files for running tests. So, it might be wise to keep them. However, if they are not needed at the moment, and all the CLI tools and Airflow workflows run perfectly without them, we might decide to at them only later, when we will need them. 

**FYI:** This is the COMPLETE list of prefixes that you might need in the transformation rules (we need to make sure that this is kept up to date):
https://github.com/meaningfy-ws/ted-sws/blob/6232c942e4e23f2c816703de233b2e4687d471a0/ted_sws/resources/prefixes/prefixes.json#L54

**Review by csnyulas — APPROVED** [review]

**Pull request merged**